.circleci/config.yml

```diff
@@ -351,6 +351,16 @@ workflows:
           requires:
             - prep-build-ts-migration-dashboard
 
+  rerun-from-failed:
+    when:
+      condition:
+        equal: ["<< pipeline.schedule.name >>", "rerun-from-failed"]
```

Comment:
this workflow will only run in develop, and if it's triggered with this exact name

![Screenshot from 2024-11-11 16-37-41](https://github.com/user-attachments/assets/9f7615df-25f3-4e38-a010-dc684aa9f540)